backend/src/v5/schemas/tickets/validators.js

```diff
@@ -104,6 +104,9 @@ Validators.propTypesToValidator = (propType, isUpdate, required) => {
 		return imposeNullableRule(Yup.array().of(types.strings.title));
 	case propTypes.IMAGE:
 		return types.embeddedImage(isNullable);
+	case propTypes.IMAGE_LIST:
+		return imposeNullableRule(Yup.array().of(isUpdate ? types.embeddedImageOrRef() : types.embeddedImage())
```

Comment:
can't we do .min(1) instead of having a weird transform rule?

Reply:
I did this because I saw this

![image](https://github.com/user-attachments/assets/903f01c7-9ce2-41e9-bd45-2b061a728fd8)

If I set min(1) it will not allow an empty array as an input isnt it ?

Reply:
yea we shouldn't I don't think